.github/workflows/test.yml

```diff
@@ -163,7 +163,11 @@ jobs:
 
       - name: Destroy AWS stacks used for testing
         run: |
-          poetry run cdk destroy --force --all
+          cdk --app cdk.out ls | while read line ; do
+             if [ "$line" != "" ] ; then
```

Comment:
As far as I can tell this isn't necessary. The empty line printed by `cdk` is on standard error, so it won't be sent to the `read` command.